gallery/src/components/NxFieldset/NxFieldsetPage.tsx

```diff
@@ -331,6 +331,37 @@ const NxFieldsetPage = () =>
                 any validation errors based on those values, and whether the checkbox group is pristine.
               </NxTable.Cell>
             </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>useTransferListState</NxCode></NxTable.Cell>
```

Comment:
Pure state helpers need docs

Reply:
> Pure state helpers need docs

Ops, Added it here: dfe965cd1e72f3801f852125b396bf5863a71c3f